### PR TITLE
Implement navbar.

### PR DESCRIPTION
**Pull request by JacksonStaniec**
### Summary 

This PR adds the navbar. 
- Created `Navbar` component
- Added `build` script to create static website for deployment
- Created gray filler space as website body placeholder for now

### Figma Page 
https://www.figma.com/file/6hSXHwtXblzSaR6rFV2b8p/SUMMER22-Website-Dev-Handoff?node-id=5%3A2

### Test Plan 

image

### CODE REVIEW

**Comment by henry-li-06** [comment]
It'd probably be a good idea to link the related Figma files in the PR description in the future.

File: dti-website-redesign/src/components/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+const Navbar = ({ hidden }: Props): JSX.Element =>
+  hidden ? (
+    <></>
+  ) : (
+    <div className={styles.navbarContainer}>
+      <>
+        <Link href="/">
+          {/* eslint-disable jsx-a11y/anchor-is-valid */}
+          <a className={styles.navLogo}>
+            <img className={styles.logo} src="/branding/wordmark.png" alt="wordmark" />
```
Comment (henry-li-06) [inline dti-website-redesign/src/components/Navbar.tsx:27]: I don't think this class is used in the module.css file?

**Comment by henry-li-06** [comment]
Not sure if I missed this in some planning meeting, but are there plans to make the navbar responsive (e.g. have a hamburger-style menu when the page width gets too small)?

**Comment by henry-li-06** [comment]
> It'd probably be a good idea to link the related Figma files in the PR description in the future.

Made a [PR](https://github.com/cornell-dti/idol/pull/308) to add a section for this

File: dti-website-redesign/src/components/Navbar.tsx (hunk 1 above)
Comment (JacksonStaniec) [reply dti-website-redesign/src/components/Navbar.tsx]: Yea I forgot that I moved it to the anchor tag. Removed now.

**Comment by JacksonStaniec** [comment]
> Not sure if I missed this in some planning meeting, but are there plans to make the navbar responsive (e.g. have a hamburger-style menu when the page width gets too small)?

We didn't talk about this, but I'd imagine that there's going to be responsive designs coming soon. This handoff was for a very preliminary version of the navbar.

**Review by henry-li-06 — APPROVED** [review]

**Pull request merged**